dapps/tests/contracts/test/another_storage_spec.js

```diff
@@ -30,7 +33,8 @@ contract("AnotherStorage", function() {
     assert.equal(result.toString(), SimpleStorage.options.address);
   });
 
-  it("set ENS address", async function() {
+  // FIXME add back when the ENS feature is back
+  xit("set ENS address", async function() {
```

Comment:
is `xit` a typo?

Reply:
No, it's to disable the test. I did so because the ENS feature needs to be re-added to tests, but it's a whole different adventure that requires it's own PR